framework/src/postprocessors/Postprocessor.C

```diff
@@ -52,3 +52,15 @@ Postprocessor::Postprocessor(const MooseObject * moose_object)
     mooseAssert(fe_problem.getReporterData().hasReporterValue<PostprocessorValue>(r_name),
                 "Postprocessor Reporter value is not declared");
 }
+
+PostprocessorValue
+Postprocessor::getValue()
+{
+  return static_cast<const Postprocessor *>(this)->getValue();
+}
+
+PostprocessorValue
+Postprocessor::getValue() const
+{
+  mooseError("getValue() (const or non-const) must be implemented.");
```

Comment:
```suggestion
  mooseError("getValue() (const or non-const, const much preferrably) must be implemented.");
```

Reply:
probably not needed if the whole fixing the herd does not drag on for 6 months

Reply:
Yeah, no one should hit this error message really.